src/diffpy/srfit/equation/builder.py

```diff
@@ -644,18 +641,19 @@ def __wrapSrFitOperators():
     diffpy.srfit.equation.literals.operators module as OperatorBuilder
     instances in the module namespace.
     """
-    import inspect
     opmod = literals.operators
     for opname in dir(opmod):
         opclass = getattr(opmod, opname)
-        if inspect.isclass(opclass) \
-            and issubclass(opclass, opmod.Operator) \
-            and opclass is not opmod.Operator \
-            and opclass is not opmod.UFuncOperator:
-
-            op = opclass()
-            _builders[op.name] = OperatorBuilder(op.name, op)
-
+        skip = (not inspect.isclass(opclass) or
+                not issubclass(opclass, opmod.Operator) or
+                inspect.isabstract(opclass) or
+                opclass is opmod.CustomOperator or
+                opclass is opmod.UFuncOperator)
+        if skip:
```

Comment:
Sure - see if 6aad8145 looks better.

BTW, when poking around that code I found that abstract classes were not correctly defined in Python 3.  I fixed that in ba3988d.